packages/babel-plugin-transform-object-rest-spread/test/fixtures/assumption-pureGetters/rest-remove-unused-excluded-keys/output.js

```diff
@@ -1,12 +1,10 @@
 const _excluded = ["excluded", "excluded2", "used", "used2"],
   _excluded2 = ["unused"];
+var _c;
 // should not remove when destructuring into existing bindings
-var _c = c2;
-({
+_c = c2, {
   a2
-} = _c);
-b2 = babelHelpers.objectWithoutProperties(_c, ["a2"]);
-_c;
+} = _c, b2 = babelHelpers.objectWithoutProperties(_c, ["a2"]);
```

Comment:
In all these cases, could we generate `_c = { a2 } = c2, b2 = babelHelpers.objectWithoutProperties(_c, ["a2"])` instead?

Reply:
In cases where the value `c2` is not read, then we could make it even smaller:
```js
b2 = babelHelpers.objectWithoutProperties({ a2 } = c2, ["a2"])
```

with no temp variable.